### ISSUE TITLE
GOAT is amazing!

### ISSUE THREAD

**Issue by pitosalas** [facebookresearch/home-robot#512]
Does this repo have GOAT code in it? Because I can't find it ...

**Comment by chrisalbertson** [comment]
I think maybe this
https://github.com/facebookresearch/home-robot/blob/ray_tracing/projects/spot/goat.py

**Comment by yvsriram** [comment]
Yes this is right. You can also check this out if you are interested in running it in sim: https://github.com/Ram81/goat-bench
